app/controllers/statistics_controller.rb

```diff
@@ -0,0 +1,98 @@
+class StatisticsController < ApplicationController
+  layout 'statistics'
+  before_action :set_commons
+
+  def notices
+    @notices_count = Notice.get_approximate_count
+    @average_count = @notices_count / ([1, Entity.count].max)
+    @notice_topics = Notice::TYPES
+  end
+
+  def infringing_urls
+    @urls_count = InfringingUrl.get_approximate_count
+    @average_count = (@urls_count / ([1, Notice.get_approximate_count].max).to_f).ceil
+  end
+
+	def entities
+		@unique_entities = Entity.count
+	end
+
+	def datewise_notices
+		render json: Notice.group_by_year(:created_at).count
+	end
+
+	def datewise_urls
+		@total_urls = InfringingUrl.get_approximate_count
+		sample_count = [ENV.fetch('URLS_GRAPH_SAMPLE_SIZE', 100000), @total_urls].min
+		sample_urls = url_sampling(sample_count)
+		tt = sample_urls.group_by_year { |u| u }
```

Comment:
It's best to have a more readable variable name.

Reply:
Done. Thank you!